csrc/keyutils.cpp

```diff
@@ -169,4 +169,36 @@ const EVP_MD* digestFromJstring(raii_env& env, jstring digestName)
 
     return result;
 }
+
+RSA* RSA_new_private_key_no_e(BIGNUM* n, BIGNUM* d)
+{
+    RSA_auto rsa = RSA_auto::from(RSA_new());
+    if (rsa.get() == nullptr) {
+        throw_openssl("RSA_new failed");
+    }
+    // The FIPS builds of AWS-LC do not have an API to disable blinding.
+#ifdef FIPS_BUILD
+
+    // RSA struct is not opaque in FIPS mode.
+    rsa->flags |= RSA_FLAG_NO_BLINDING;
+
+#else
+
+    // RSA_blinding_off_temp_for_accp_compatibility is marked deprecated, so we need to disable the
+    // "deprecated-declarations" check for this invocation.
+#pragma GCC diagnostic push
+#pragma GCC diagnostic ignored "-Wdeprecated-declarations"
+    // We need to change this API once AWS-LC provides a method similar to the following:
+    // https://github.com/google/boringssl/blob/master/include/openssl/rsa.h#L630
+    RSA_blinding_off_temp_for_accp_compatibility(rsa);
```

Comment:
should we file an Issue against AWS-LC for a stable API to accomplish this?

Reply:
AWS-LC has this on their radar.